docs/guide/test-id.mdx

```diff
@@ -3,30 +3,37 @@ import TabItem from '@theme/TabItem';
 import Android from './partials/_find-test-id-android.mdx';
 import IOS from './partials/_find-test-id-ios.mdx';
 
-# Adding testID to your components
+# Adding test ID's to your components
 
 :::info Note
 
-This guide is applicable only for React Native applications.
+This guide is optimized for React Native apps, but can be generalized for testing any app, including native apps.
 
 :::
 
-It is always the best idea to match your element by something unique. We recommend using [`testID` property](https://reactnative.dev/docs/view#testid) supported by most React Native components:
+While [view-element matching](../api/matchers.md) can be done in numerous ways, it is always the best idea to match based on something unique and decoupled, as it ensures that the test code would be clear, stable and sustainable over time. We recommend assigning unique test ID's to the elements you're aiming to interact with in your tests, and prefering matching based on those rather than on anything else. Test ID's are the least likely to change over time (compared with raw text, for example), and are locale-agnostic.
```

Comment:
IMO there's another point for test-ids:

```suggestion
While [view-element matching](../api/matchers.md) can be done in numerous ways, it is always the best idea to match based on something unique and decoupled, as it ensures that the test code would be clear, stable and sustainable over time. 

We recommend assigning unique test ID's to the elements you're aiming to interact with in your tests, and preferring matching based on those rather than on anything else. Test ID's are the least likely to change over time (compared with raw text, for example), and are locale-agnostic. Additionally, utilizing unique test IDs across the app not only simplifies the identification and interaction with specific elements but also enhances code navigability, making it easier to locate elements when traversing the codebase.
```